requirements.txt

```diff
@@ -0,0 +1 @@
+requests>=2.2.1
```

Comment:
Theoretically we could add `requirements.txt` and just populate the `install_requires` list from the file's lines when setup is run. I feel like it's a pretty ubiquitous feature of python libraries at this point. That being said, there are so few requirements, it's kind of silly having it.